src/env.js

```diff
@@ -1,8 +1,5 @@
-// Simple wrapper exposing environment variables to rest of the code.
-
 import jetpack from 'fs-jetpack'
 
-// The variables have been written to `env.json` by the build process.
-var env = jetpack.cwd(__dirname).read('env.json', 'json')
+const env = jetpack.cwd(__dirname).read('env.json', 'json')
```

Comment:
I don't think it's a good idea to make this a constant. The environment may be changed on runtime.